internal/tiltfile/k8s.go

```diff
@@ -30,7 +30,11 @@ type k8sResource struct {
 	// The name of this group, for display in the UX.
 	name string
 
-	// All k8s resources to be deployed.
+	// The single workload entity for this resource (only applies to v2 resource assembly)
```

Comment:
can you add some comments on why you broke out the data model this way? i spent a long time looking at this and couldn't figure it out.